mage_ai/shared/environments.py

```diff
@@ -16,10 +22,14 @@ def is_dev():
     return os.getenv('ENV', None) == 'dev' or os.getenv('ENV', None) == 'development'
 
 
-def is_test():
+def is_test_mage():
```

Comment:
It'll be better to add comments to both `is_test_mage` and `is_test` methods so that we know the difference. It's easy to forget the context why we need both.